packages/tab-bar/index.tsx

```diff
@@ -47,7 +47,7 @@ class TabBar extends React.Component<
     this.foundation.init();
     const {activeIndex, indexInView} = this.props;
     if (this.tabList[activeIndex]) {
-      this.tabList[activeIndex].activate({} /* previousIndicatorClientRect */);
+      this.tabList[activeIndex].activate(new DOMRect() /* previousIndicatorClientRect */);
```

Comment:
This is actually not IE11 compatible. I also realize we currently don't work on IE11, but we have some issues to be IE11 compatible. Please just change this to:

```
// new DOMRect is not IE11 compatible
const defaultDOMRect = {
bottom: 0
height: 0
left: 0
right: 0
top: 0
width: 0
x: 0
y: 0
};
...activate(defaultDOMRect);
```

You may need to cast it.